### PR TITLE
Feature/bp validation fixes

### PR DESCRIPTION
**Pull request by brianpos**
#FHIR-41652 Remove the que-14 invariant from the base questionnaire as that is only valid in R5
#FHIR-43242 Correct invalid sdc-t1 invariant
#FHIR-43243 Lots of invalid fhirpath expressions
#FHIR-43244 Missing fhirpath expression for definition search parameter
FHIR-43243 behaviour and examples narrative content also has the same valueDecimal => value issue to resolve
Fix FSH that was over-writing the variable extension rather than appending instances to the collection
FHIR-43243 More invalid fhirpath expressions valueXXX => value
FHIR-43245 Invalid fhirpath expression
FHIR-40623 typo
FHIR-40621 Cloned text removed

### CODE REVIEW

**Review by lmckenzi — COMMENTED** [review]
You added this into a bunch of examples:
```
* extension[0]
  * url = $standard-status
  * valueCode = #draft
* extension[+]
  * url = $fmm
  * valueInteger = 0
 ``` 
Why?  Standards status should automatically get set to 'informative' for examples, and FMM level isn't relevant...
  
You also changed:
`* extension[$variable]`
to
`* extension[+]
  * url = $variable`

Why?

Fine with everything else.

**Comment by brianpos** [comment]
It wasn't added, it was moved. It was already in the resource, just in the other format.
Can you tag a comment on the specific ones you're concerned on?

File: input/fsh/instances/questionnaire-sdc-profile-example-framingham-hchd-lhc.fsh [hunk 1]
```diff
@@ -3,83 +3,102 @@ InstanceOf: SDCBaseQuestionnaire
 Title: "SDC-Hard Coronary Heart Disease"
 Description: "A made-up form for calculating the 10-year-risk of Hard Coronary Heart Disease. It shows how variables and calculatedExpression can be used to compute for the risk probability."
 Usage: #example
-* extension[$variable]
+* extension[0]
```
Comment (lmckenzi) [inline input/fsh/instances/questionnaire-sdc-profile-example-framingham-hchd-lhc.fsh:6]: Why change the format of declaring the extensions?

File: input/fsh/instances/questionnaire-sdc-profile-example-hunger-vital-signs.fsh [hunk 2]
```diff
@@ -2,33 +2,42 @@ Instance: questionnaire-sdc-profile-example-hunger-vital-signs
 InstanceOf: SDCBaseQuestionnaire
 Title: "SDC-Hunger Vital Sign"
 Usage: #example
-* extension[$variable]
+* extension[0]
```
Comment (lmckenzi) [inline input/fsh/instances/questionnaire-sdc-profile-example-hunger-vital-signs.fsh:5]: And here

File: input/fsh/instances/questionnaire-sdc-profile-example-framingham-hchd-lhc.fsh (hunk 1 above)
Comment (brianpos) [reply input/fsh/instances/questionnaire-sdc-profile-example-framingham-hchd-lhc.fsh:6]: As when using the equals one, they just blat over any others, so you just end up with 1 variable in the output!!!

**Review by lmckenzi — APPROVED** [review]

**Pull request merged**